client.lua

```diff
@@ -163,199 +199,106 @@ local function startTestDriveTimer(testDriveTime, prevCoords)
     end)
 end
 
+local function enteringVehicleSellZone()
+    if PlayerData and PlayerData.job and (PlayerData.job.name == Config.Shops[insideShop]['Job'] or Config.Shops[insideShop]['Job'] == 'none') then
+        lib.showTextUI(Lang:t('menus.keypress_vehicleViewMenu'))
+    end
+end
+
+local function insideVehicleSellZone()
+    if IsControlJustPressed(0, 38) then
+        if PlayerData and PlayerData.job and (PlayerData.job.name == Config.Shops[insideShop]['Job'] or Config.Shops[insideShop]['Job'] == 'none') then
+            openVehicleSellMenu()
+        end
+    end
+end
+
+local function exitingVehicleSellZone()
+    lib.hideTextUI()
+end
```

Comment:
This function seems pretty useless, taking up space where it is not needed

Reply:
Sadly i don't fully understand it :'D But i did a minor change in the next commit